app/controllers/api/v1/bookshelf_books_controller.rb

```diff
@@ -0,0 +1,16 @@
+class Api::V1::BookshelfBooksController < ApplicationController
+  def index
+    books = BookshelfFacade.books(bookshelf_books_params)
+    render json: BookSerializer.list_books(books), status: :ok
+  end
+
+  private
+
+  def bookshelf_books_params
+    bookshelf_books_params = {
+      uid: params[:user_id],
```

Comment:
Fab! The mock requests and responses in the Postman docs look great also. Thank you!